Adapt MediaRecorder API to new events structure (#13850)
mdn/content: files/en-us/web/api/mediarecorder/stop_event/index.md

>   - Property

Reviewer: ```suggestion
  - Event
```
Or it won't be listed correctly in the sidebar.

  - Reference
  - Video
  - stop
browser-compat: api.MediaRecorder.stop_event
---
{{APIRef("Media Recorder API")}}

> The `stop` event is thrown either as a result of the
> {{domxref("MediaRecorder.stop()")}} method being invoked, or when the media stream being

Reviewer 2: ```suggestion
The `stop` event is fired when
{{domxref("MediaRecorder.stop()")}} is called, or when the media stream being
```
